hat/assets/js/apps/Iaso/domains/users/permissionsGroupsMessages.ts

```diff
@@ -48,6 +48,10 @@ const PERMISSIONS_GROUPS_MESSAGES = defineMessages({
         id: 'iaso.permissions.group.admin',
         defaultMessage: 'Admin:',
     },
+    registry: {
+        id: 'iaso.permissions.group.registry',
+        defaultMessage: 'Registry:',
```

Comment:
The colon `:` in the `defaultMessage` should be removed. I think it's an error in the other messages as well, because it deosn't match the translation in the json files. So it should either be removed from the `defaultMessage` or added to the json translation. 
The existing ones should be updated as well, but it's not necessarily in the scope of the PR